src/Get.test.tsx

```diff
@@ -65,6 +65,89 @@ describe("Get", () => {
       await wait(() => expect(children.mock.calls.length).toBe(2));
     });
 
+    it("should compose absolute urls", async () => {
+      nock("https://my-awesome-api.fake")
+        .get("/people")
+        .reply(200);
+
+      nock("https://my-awesome-api.fake")
+        .get("/absolute")
+        .reply(200);
+
+      const children = jest.fn();
+      children.mockReturnValue(<div />);
+
+      render(
+        <RestfulProvider base="https://my-awesome-api.fake">
+          <Get path="/people">{() => <Get path="/absolute">{children}</Get>}</Get>
+        </RestfulProvider>,
+      );
+
+      await wait(() => expect(children.mock.calls.length).toBe(3));
+    });
+
+    it("should compose relative urls", async () => {
+      nock("https://my-awesome-api.fake")
+        .get("/people")
+        .reply(200);
+
+      nock("https://my-awesome-api.fake")
+        .get("/people/relative")
+        .reply(200);
+
+      const children = jest.fn();
+      children.mockReturnValue(<div />);
+
+      render(
+        <RestfulProvider base="https://my-awesome-api.fake">
+          <Get path="/people">{() => <Get path="relative">{children}</Get>}</Get>
+        </RestfulProvider>,
+      );
+      await wait(() => expect(children.mock.calls.length).toBe(3));
+    });
+
+    it("should compose more nested absolute and relative urls", async () => {
+      nock("https://my-awesome-api.fake")
+        .get("/absolute-1")
+        .reply(200);
+
+      nock("https://my-awesome-api.fake")
+        .get("/absolute-1/relative-1")
+        .reply(200);
+
+      nock("https://my-awesome-api.fake")
+        .get("/absolute-2")
+        .reply(200);
+
+      nock("https://my-awesome-api.fake")
+        .get("/absolute-2/relative-2")
+        .reply(200);
+
+      nock("https://my-awesome-api.fake")
+        .get("/absolute-2/relative-2/relative-3")
+        .reply(200);
+
+      const children = jest.fn();
+      children.mockReturnValue(<div />);
+
+      render(
+        <RestfulProvider base="https://my-awesome-api.fake">
+          <Get path="/absolute-1">
+            {() => (
+              <Get path="relative-1">
+                {() => (
+                  <Get path="/absolute-2">
+                    {() => <Get path="relative-2">{() => <Get path="relative-3">{children}</Get>}</Get>}
+                  </Get>
+                )}
+              </Get>
+            )}
+          </Get>
+        </RestfulProvider>,
+      );
+      await wait(() => expect(children.mock.calls.length).toBe(6));
```

Comment:
Can you also add some other expects with some data from nock to be sure that we have that we are expected here? I want to know if the `.get("/absolute-2/relative-2/relative-3")` is well called